data/ecosystems/a/avalanche.toml

```diff
@@ -91,6 +91,7 @@ sub_ecosystems = [
   "Nereus Finance",
   "NFTrade",
   "Niftyx",
+  "NodeKit",
```

Comment:
I don't think we have an ecosystem called NodeKit, can you review please?

Reply:
Hey so if I am building on the Avalanche ecosystem do I add a new ecosystem and also keep the information I put here?

Reply:
That is correct, you can create an ecosystem called say `nodekit.toml` under `data/ecosytems/n` and with `title = "NodeKit"`, and then you would keep the current line here